go/executor/pai.go

```diff
@@ -466,7 +466,33 @@ func executeOptimizeUsingOptFlow(pythonExecutor *pythonExecutor, stmt *ir.Optimi
 	if err != nil {
 		return err
 	}
-	defer dropTmpTables([]string{tableName}, pythonExecutor.Session.DbConnStr)
+
+	dropTmpTableFunc := func(table string) {
+		dropTmpTables([]string{table}, pythonExecutor.Session.DbConnStr)
+	}
+
+	if len(stmt.Variables) > 1 {
+		joinedVarName := strings.Join(stmt.Variables, "__")
+		concatColumnNames := make([]string, 0)
+		for i, v := range stmt.Variables {
+			if i >= 1 {
+				concatColumnNames = append(concatColumnNames, `','`)
+			}
+			concatColumnNames = append(concatColumnNames, v)
+		}
+		concatColumnExpr := fmt.Sprintf("CONCAT(%s) AS %s", strings.Join(concatColumnNames, ","), joinedVarName)
```

Comment:
Why need a column stores the concated variable names?

Reply:
It is because the OptFlow HTTP interface only supports one column as the variable.